README.md

```diff
@@ -41,6 +41,9 @@ app locally. Do this like so:
     cd yari
     yarn install
 
+See the [troubleshooting](#troubleshooting) section below if you run into
+problems.
+
```

Comment:
I'm actually not sure we should mention this here:

```suggestion
```

Do you know if there is a way to display a TOC in the GitHub README?

Reply:
Added note: one of the markdown VSCode plugins used for /content/ includes a setting that identifies and basically rewrites TOCs. This can be considered a feature or a bug. If you start creating a TOC, it will eventually identify it as such. It will then automatically generate a markdown TOC for you. That is what you were looking for... so there is an extension to VS code that you likely have installed that does that. I assume there may even be a way to ask it to create the TOC without first having to start typing it in.

The bug part: if you want to style it your own way, not include EVERYTHING in your TOC, or in any way alter it from the plugin's preference, it rewrites it to their preference.  I used it for a TOC for content within an H2 on a long document and had to toggle the VS Code extension off to enable removing the subheadings from other sections. 

As this is a TOC for a README file, the extension is probably good as is. If you alter the extension for /content/, where we use underscores instead of dashes between words in anchor links, you would then have to turn the extension off, after the TOC is created, to switch between underscores and dashes without having the plugin "fix" it for you.

I hope that helps.

Reply:
> I'm actually not sure we should mention this here:

We have quite a long readme, I personally missed it while reading the installation instructions. TOC wouldn’t help, either. I’d keep this note for people like me 😄